src/test/sighash_tests.cpp

```diff
@@ -135,14 +135,6 @@ void static RandomTransaction(CMutableTransaction &tx, bool fSingle) {
             randombytes_buf(pourtx.ciphertexts[1].begin(), pourtx.ciphertexts[1].size());
             pourtx.macs[0] = GetRandHash();
             pourtx.macs[1] = GetRandHash();
-            {
-                std::vector<unsigned char> txt;
-                int prooflen = insecure_rand() % 1000;
-                for (int i = 0; i < prooflen; i++) {
-                    txt.push_back(insecure_rand() % 256);
-                }
-                pourtx.proof = std::string(txt.begin(), txt.end());
-            }
```

Comment:
Should this be replaced by a call to `randombytes_buf()` filling the proof array?